start.sh

```diff
@@ -41,10 +64,11 @@ MEM_USED=$(($MEM_USED+0))
 # Si la mémoire max est dépassée
 if (( $MEM_USED > $MAX_RAM ))
 then
-    kill -9 $(pgrep mitmdump) > /dev/null
-    echo "Mitmdump arrêté"
+    kill -9 $PID
+    wait $PID &> /dev/null
+    echo $CMD_NAME" arrété"  
 fi
-sleep 1
+sleep 2
 done
 done
```

Comment:
ce script ressemble de plus en plus a un script d'init.

Regarde:

https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files